roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -308,6 +308,8 @@ expand_persistent_volumes: false
 ## When OpenStack is used, Cinder version can be explicitly specified if autodetection fails (Fixed in 1.9: https://github.com/kubernetes/kubernetes/issues/50461)
 # openstack_blockstorage_version: "v1/v2/auto (default)"
 openstack_blockstorage_ignore_volume_az: "{{ volume_cross_zone_attachment | default('false') }}"
+# set max volumes per node (cinder-csi), default not set
+# node_volume_attach_limit: 25
```

Comment:
Since this affects only the Cinder CSI, should we prefix this variable with `openstack_` or `cinder_`?

Reply:
When this variable was implemented for the old Openstack provider (https://github.com/kubernetes-sigs/kubespray/pull/5089) there was a discussion for having a variable that could be used for other cloud providers as well.

I just took an existing variable and reused it in this code.

I can rename the variable to something else. I assume cinder_ would be the go-to in that case?

Reply:
Oh right I didn't notice that this variable was there in the in-tree provider as well: https://github.com/kubernetes-sigs/kubespray/blob/4dba34bd0239b36b9132bb38257c1feca469c23b/roles/kubernetes/node/templates/cloud-configs/openstack-cloud-config.j2#L31

Then I don't mind the name in this PR. 

Thanks for the refresher!

Reply:
Hi @bl0m1
I used the `cinder_` prefix in all variables in my Cinder CSI PR so that people could configure both the in-tree OpenStack and CSI driver before the in-tree is totally deprecated and removed.
So maybe `node_volume_attach_limit` should also be prefixed with `cinder_` if it is intended to be used with the Cinder CSI driver?